app/perf.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title>popgun demo</title>
+    <link rel="stylesheet" type="text/css" href="app.css">
+
+    <!-- merchants -->
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,400italic,700,700italic,900,900italic,300italic,300,100italic,100' rel='stylesheet' type='text/css'>
+    <style type='text/css' href='app.css'></style>
+    <style>
+    .target {
```

Comment:
I don't think you use most of these classes, so feel free to remove w/e you don't need.